be/src/vec/sink/writer/vtablet_writer.cpp

```diff
@@ -1786,22 +1792,27 @@ Status VTabletWriter::append_block(doris::vectorized::Block& input_block) {
                 return Status::NeedSendAgain("");
             }    // creating done
         } else { // not auto partition
-            for (int i = 0; i < num_rows; ++i) {
-                if (UNLIKELY(has_filtered_rows) && _block_convertor->filter_map()[i]) {
-                    continue;
+            std::vector<VOlapTablePartition*> partitions {num_rows, nullptr};
+            std::vector<bool> skip;
+            skip.resize(num_rows);
```

Comment:
why not call call contruct like partitions，no need call `resize`

Reply:
`vector<bool>` is not same as other vector